source/css/_schemes/Pisces/_sidebar.styl

```diff
@@ -119,3 +119,11 @@
     }
   }
 }
+
+.back-to-top {
```

Comment:
Emm ... Now the style when `back2top.sidebar` is false is also affected

<img width="285" alt="截屏2020-02-13下午12 25 32" src="https://user-images.githubusercontent.com/16272760/74401402-171dad00-4e5c-11ea-8154-b64d1c39c56f.png">

This issue is very tricky because the styles of the various parts of the theme NexT are completely mixed together